File: rust/src/models/boolean_config_value.rs
// this file is @generated
use serde::{Deserialize, Serialize};

/// A boolean config value.
#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct BooleanConfigValue {
    pub value: bool,
}

impl BooleanConfigValue {
    pub fn new(value: bool) -> Self {
        Self { value }
    }
}

File: typescript/src/models/booleanConfigValue.ts
// this file is @generated
/** A boolean config value. */
export interface BooleanConfigValue {
  value: boolean;
}

export const BooleanConfigValueSerializer = {
  _fromJsonObject(object: any): BooleanConfigValue {
    return {
      value: object["value"],
    };
  },

  _toJsonObject(self: BooleanConfigValue): any {
    return {
      value: self.value,
    };
  },
};
